src/platform_impl/apple/appkit/view.rs

```diff
@@ -1029,6 +1042,12 @@ impl WinitView {
     }
 
     fn mouse_click(&self, event: &NSEvent, button_state: ElementState) {
+        if !self.ivars().focusable.get() {
+            let mtm = MainThreadMarker::from(self);
+            unsafe {
+                NSApplication::sharedApplication(mtm).preventWindowOrdering();
```

Comment:
Nit: It'd be nice with a comment here detailing why `preventWindowOrdering`, an application-level method, is fine to call inside `NSView`.